frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductModal.tsx

```diff
@@ -92,20 +92,25 @@ export const ProductModal = ({
       // Validate that it is a money value.
       // Regex allows leading and trailing spaces, max 2dp
       const validateMoney = /^\s*(\d+)(\.\d{0,2})?\s*$/.test(val ?? '')
-      if (!validateMoney) return 'Please enter a valid payment amount'
+      if (!validateMoney)
+        return `Enter an amount between S${formatCurrency(
+          Number(centsToDollars(minPaymentAmountCents)),
+        )} and S${formatCurrency(
+          Number(centsToDollars(maxPaymentAmountCents)),
+        )}`
 
       const validateMin = !!val && dollarsToCents(val) >= minPaymentAmountCents
       // Repeat the check on minPaymentAmountCents for correct typing
       if (!!minPaymentAmountCents && !validateMin) {
-        return `The maximum amount is ${formatCurrency(
+        return `The maximum amount is S${formatCurrency(
```

Comment:
Typo that wasn't caught in previous design reviews / QAs? 👀 

```suggestion
        return `The minimum amount is S${formatCurrency(
```